src/execute/engine.ts

```diff
@@ -143,6 +147,38 @@ export function markdownExecutionEngine(markdown: string, flags?: RenderFlags) {
   return markdownEngine;
 }
 
+function reorderEngines(project: ProjectContext) {
+  // TODO: add `engines` to schema
```

Comment:
This is done by [`8e76196` (#11807)](https://github.com/quarto-dev/quarto-cli/pull/11807/commits/8e761969f58a27d66e96758d73901dc10d1ca61e) right ?

Reply:
ah then that's why it had no effect, I will remove the todo comment